website/docs/r/server.html.md

```diff
@@ -229,6 +233,7 @@ This resource exports the following attributes:
 * `location_uuid` - The location this server is placed. The location of a resource is determined by it's project.
 * `labels` - List of labels in the format [ "label1", "label2" ].
 * `hardware_profile` - The hardware profile of the server.
+* `user_data_base64` - See Argument Reference above.
```

Comment:
`user_data_base64` will always be returned regardless what the customer uses user_data_base64 or user_data